_config.yml

```diff
@@ -65,3 +67,4 @@ kramdown:
     block:
       css_class: code_section
 
+repository: crystal-lang/crystal-website
```

Comment:
What is this for? It's unrelated to this PR, right?

Reply:
It's used in the `component` include to generate links to the source code.

https://github.com/straight-shoota/crystal-website/blob/b88a3177e52dbb946c01db5033dbdd39f6521457/_includes/style-guide/component.html#L5-L6